db/r2retdec

```diff
@@ -28,7 +19,7 @@ R2PM_INSTALL() {
 		-DBUILD_BUNDLED_RETDEC=ON \
 		-DRETDEC_INSTALL_PREFIX="${PWD}/install" || exit 1
 	fi
-	${MAKE} -j$(fetch_nproc) install || exit 1
+	${MAKE} -j$(nproc --all) install || exit 1
```

Comment:
I think is better to force 2. In my host using -j the compilation takes 24GB of ram. So better kill that nproc completely